geopandas/sindex.py

```diff
@@ -81,22 +250,135 @@ def is_empty(self):
 
 if compat.HAS_PYGEOS:
 
-    from pygeos import STRtree, box, points  # noqa
+    from . import geoseries  # noqa
+    from .array import GeometryArray, _shapely_to_geom  # noqa
+    import pygeos  # noqa
+    from pygeos import STRtree, box, points, Geometry  # noqa
```

Comment:
I don't think `Geometry` is actually used below? 

Also, maybe we should just use the `import pygeos` for all usage of pygeos, that might be more explicit

Reply:
Will do, I agree.